latest/src/app/documentation/demos/datagrid/server-driven/examples.ts

```diff
@@ -5,17 +5,17 @@
  */
 export const EXAMPLES = {
     stateInterface: `
-interface ClrDatagridStateInterface {
+interface ClrDatagridStateInterface<T = any> {
     page?: {
         from?: number;
         to?: number;
         size?: number;
     }
     sort?: {
-        by: string | Comparator<any>
+        by: string | ClrDatagridComparatorInterface<T>
```

Comment:
Do our formatters work in the `website` branch? Looks as if some properties are missing `;` here.

Reply:
oh i don't think we run a formatter on the website branch. I can add the `;`s though.